.pipelines/azure-pipeline-build.yml

```diff
@@ -345,7 +345,7 @@ jobs:
 
         docker buildx create --name dockerbuilder
         docker buildx use dockerbuilder
-        docker buildx build . --platform=linux/amd64,linux/arm64 --file ./build/linux/Dockerfile -t $(LINUX_FULL_IMAGE_NAME) --metadata-file $(Build.ArtifactStagingDirectory)/linux/metadata.json --push
+        docker buildx build . --platform=linux/amd64,linux/arm64 --file ./build/linux/Dockerfile -t $(LINUX_FULL_IMAGE_NAME) --metadata-file $(Build.ArtifactStagingDirectory)/linux/metadata.json --push --cache-to type=registry,ref=$(ACR_REGISTRY)$(ACR_REPOSITORY)/cache:prometheuscollector,mode=max --cache-from type=registry,ref=$(ACR_REGISTRY)$(ACR_REPOSITORY)/cache:prometheuscollector
```

Comment:
Hi Grace - does this help much ? Because every build creates a  different manifest (due to changes in our layer), which cannot be reused by future builds. Isn't it ? Also, when these build tasks are done in parallel, they cannot be used across each, isn't it ?

I see there is no MCR reconciliation for cache, bit will we be flagged by s360 for stale manifests in our ACR (the reason due to which we currently clean up in 2 hours...)

Reply:
Thanks Grace.